e2e/util_test.go

```diff
@@ -368,15 +498,120 @@ func replaceMatchWithSpace(input string, match ...string) string {
 	return input
 }
 
-func s3BucketFromTestName(t *testing.T) string {
+func s3BucketFromTestNameWithPrefix(t *testing.T, prefix string) string {
 	t.Helper()
 	bucket := strcase.ToKebab(t.Name())
 
-	if len(bucket) > 63 {
-		bucket = fmt.Sprintf("%v-%v", bucket[:55], randomString(7))
+	reg, _ := regexp.Compile("[^-a-z0-9]+")
+	prefix = reg.ReplaceAllString(prefix, "")
```

Comment:
Second `ReplaceAllString` call a couple of line below should do the job. Is this line necessary?

Reply:
I agree, I will remove the first `ReplaceAllString`